test/parallel/test-icu-data-dir.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
-require('../common');
+const common = require('../common');
+if (!(common.hasIntl && common.hasSmallICU)) {
```

Comment:
Good catch! I was looking at an old [commit](https://github.com/nodejs/node/pull/12226/commits/a9d72e1a5d4f390abaf5d78384eb9508dec010c6) and did not notice that that function does not exist or has been removed. Going to add it now. Thanks.